docs/public-networks/reference/api/index.md

```diff
@@ -2775,6 +2775,179 @@ curl -X POST -H "Content-Type: application/json" --data '{ "query": "{block (num
 
 <!--/tabs-->
 
+### `eth_getBlockReceipts`
+
+Returns all transaction receipts for a given block. Transaction receipts provide a way to track the success or failure of a transaction (`1` if successful and `0` if failed), as well as the amount of
+gas used and any event logs that might have been produced by a smart contract during the transaction.
+
+#### Parameters
+
+- `blockNumber`: _string_ - hexadecimal or decimal integer representing a block number or one of the string tags `latest`, `earliest`, `pending`, `finalized`, or `safe`, as described in [block parameter](../../how-to/use-besu-api/json-rpc.md#block-parameter)
+
+- `verbose`: _boolean_ - if `true`, returns the full [transaction objects](objects.md#transaction-object); if `false`, returns only the hashes of the transactions.
```

Comment:
`verbose` is not one of the parameters for this method, it's just the block parameter: https://github.com/ethereum/execution-apis/blob/057892b256c181232ca22499d1801b9637192698/src/eth/block.yaml#L89-L95

Reply:
Block parameter has been incorporated.